tun_linux.go

```diff
@@ -474,6 +474,26 @@ func (t *NativeTun) rules() []*netlink.Rule {
 	}
 
 	nopPriority := ruleStart + 10
```

Comment:
Should the +10 here be changed to +11?

Reply:
the 10 is keep for other rules.
if rule's length greater than 10, the nopPriority will be overlapped.
it shouldn't be hardcode by 10.

Reply:
You added a port rule here, so you should change +10 to +11 to ensure that there is no overlap.
Hard coding may not be the best solution, but it does work.

Reply:
Fixed